gui/pages/Content/Models/AddModelMarketPlace.js

```diff
@@ -0,0 +1,103 @@
+import React, {useState, useEffect} from "react";
+import Image from "next/image";
+import {openNewTab, modelIcon} from "@/utils/utils";
+import {fetchApiKey, storeModel} from "@/pages/api/DashboardService";
+import {toast} from "react-toastify";
+
+export default function AddModelMarketPlace(template){
+    const [modelTokenLimit, setModelTokenLimit] = useState(4096);
+    const [modelVersion, setModelVersion] = useState('');
+    const [modelEndpoint, setModelEndpoint] = useState('');
+    const [tokenError, setTokenError] = useState(false);
+    const [templateData, setTemplateData] = useState(template.template);
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(()=>{
+        console.log(templateData)
```

Comment:
remove log statements

Reply:
Sir console.log will be removed once the QA testing is done, I am keeping it in case of any potential bugs and to see the output from the api